src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -338,6 +346,42 @@ export class EditConcept extends Component {
     });
   }
 
+  handleAsyncSelectChange = (answers, uniqueKey) => {
+    const { addSelectedAnswers, selectedAnswers } = this.props;
+    addSelectedAnswers(answers, uniqueKey);
+
+    const answersToAdd = selectedAnswers
+      .filter(ans => ans.map_type === 'Q-AND-A')
```

Comment:
Should we hard code this?

Reply:
@dkayiwa let me create a variable for this.